### PR TITLE
Update LOOT-FAQs.html

### PR DESCRIPTION
**Pull request by pStyl3**
Differentiate between `URL` and `File Path`.

### CODE REVIEW

**Comment by pStyl3** [comment]
If there aren't any objections to this, I'd like to merge this change tomorrow.

File: docs/help/LOOT-FAQs.html [hunk 1]
```diff
@@ -129,7 +129,7 @@ <h3>Common Issues</h3>
...
 
-    <p>If LOOT still can't update its masterlist, you can update manually by downloading <a href="https://raw.githubusercontent.com/loot/skyrim/v0.15/masterlist.yaml">this file</a> and placing it in the <code>%LOCALAPPDATA%\LOOT\Skyrim</code> folder. Replace <code>skyrim</code> with <code>morrowind</code>, <code>oblivion</code>, <code>fallout3</code>, <code>falloutnv</code>, <code>fallout4</code> or <code>skyrimse</code> as necessary in the URL and the folder path.</p>
+    <p>If LOOT still can't update its masterlist, you can update manually by downloading <a href="https://raw.githubusercontent.com/loot/skyrim/v0.15/masterlist.yaml">this file</a> and placing it in the <code>%LOCALAPPDATA%\LOOT\Skyrim</code> folder. Replace <code>skyrim</code> within the URL with <code>morrowind</code>, <code>oblivion</code>, <code>skyrimse</code>, <code>fallout3</code>, <code>falloutnv</code>, <code>fallout4</code> or <code>enderal</code> as necessary, and <code>Skyrim</code> within the folder path with <code>Morrowind</code>, <code>Oblivion</code>, <code>Skyrim Special Edition</code>, <code>Fallout3</code>, <code>FalloutNV</code>, <code>Fallout4</code> or <code>Enderal</code>.</p>
```
Comment (sibir-ine) [inline docs/help/LOOT-FAQs.html:132]:
```suggestion
    <p>If LOOT still can't update its masterlist, you can update manually by downloading <a href="https://raw.githubusercontent.com/loot/skyrim/v0.15/masterlist.yaml">this file</a> and placing it in the <code>%LOCALAPPDATA%\LOOT\Skyrim</code> folder. Depending on your game, replace <code>skyrim</code> within the URL with <code>morrowind</code>, <code>oblivion</code>, <code>skyrimse</code>, <code>fallout3</code>, <code>falloutnv</code>, <code>fallout4</code> or <code>enderal</code> as necessary, and <code>Skyrim</code> within the folder path with <code>Morrowind</code>, <code>Oblivion</code>, <code>Skyrim Special Edition</code>, <code>Skyrim VR</code>, <code>Fallout3</code>, <code>FalloutNV</code>, <code>Fallout4</code>, <code>Fallout4VR</code>, <code>Nehrim</code>, <code>Enderal</code> or <code>Enderal Special Edition</code>.</p>
```

**Review by Ortham — CHANGES_REQUESTED** [review]
Sorry for the delay, I'm fine with this if sibir's suggested change is incorporated.

**Comment by pStyl3** [comment]
Done, thanks for the feedback.

**Pull request merged**